pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -36,12 +37,189 @@ import (
 	"github.com/vmware-tanzu/antrea/pkg/ovs/ovsconfig"
 )
 
+// NetDeviceType type Enum
+const (
+	netDeviceTypeVeth = "veth"
+	netDeviceTypeVF   = "vf"
+)
+
 type ifConfigurator struct {
-	ovsDatapathType string
+	ovsDatapathType      string
+	isOVSHardwareOffload bool
```

Comment:
please rename to `isOVSHardwareOffloadEnabled`

Reply:
done